package.json

```diff
@@ -46,9 +46,11 @@
     "apollo-client": "^2.6.10",
     "apollo-link-http": "^1.5.17",
     "cids": "^1.0.0",
+    "commitizen": "^4.2.4",
```

Comment:
In the future @rej156  let's put this stuff in a different PR.

It's best to have PRs that have a "single-thesis", so that it's easier to review the code and roll back in case something breaks

Reply:
generally agree but naughtily added it in there cause it supports what you just said
commit message wizard formatter to help for rollback

will make nice to haves in separate prs though for future notice 🙇 senpai